package hva.core.exception;

public class ResponsabilityNotThereException extends Exception {

    public ResponsabilityNotThereException(String idRes) {
        super("Responsabilidade n√£o existe" + idRes);
    }

    public ResponsabilityNotThereException(Throwable cause) {
        super(cause);
    }
    
}
